packages/quantic/docs/template/lwc-json/publish.js

```diff
@@ -225,6 +225,7 @@ exports.publish = async function (data, opts) {
   if (opts.destination === 'console') {
     console.log(dump(root));
   } else {
+    fs.mkdirSync(resolve(dirname(opts.destination)), {recursive: true});
```

Comment:
is `fs.mkdirSync` safe if ever the folder already exists?

Reply:
Yes 👍 with `recursive: true`, mkdirSync does not throw if the directory exists

> Calling fs.mkdir() when path is a directory that exists results in an error only when recursive is false

https://nodejs.org/docs/latest/api/fs.html#fsmkdirpath-options-callback